addons/repair/functions/fnc_addRepairActions.sqf

```diff
@@ -96,7 +89,28 @@ _wheelHitPointSelections = _wheelHitPointsWithSelections select 1;
         };
 
         _icon = "A3\ui_f\data\igui\cfg\actions\repair_ca.paa";
+
         _selection = "";
+        // Get custom position if available
+        _customSelectionsCfg = configFile >> "CfgVehicles" >> _type >> QGVAR(hitpointPositions);
+        if (isArray _customSelectionsCfg) then {
```

Comment:
This code block is hard to read. Please write more comments saying what's going on. Though I suspect that this will change with a different format in the config.